internal/filter/filter.go

```diff
@@ -228,6 +228,19 @@ func (f *Filter) EnqueueAllPreflightValidations(mod client.Object) []reconcile.R
 	return reqs
 }
 
+func (f *Filter) EnqueueAllPreflightValidationsOCP(pv client.Object) []reconcile.Request {
+	logger := f.logger.WithValues("preflight", pv.GetName())
+	pvo := kmmv1beta1.PreflightValidationOCP{}
+
+	nsn := types.NamespacedName{Name: pv.GetName(), Namespace: pv.GetNamespace()}
+	err := f.client.Get(context.Background(), nsn, &pvo)
+	if err != nil {
+		logger.Error(err, "failed to find PVO related to", "preflight", nsn)
+		return []reconcile.Request{}
+	}
+	return []reconcile.Request{reconcile.Request{NamespacedName: nsn}}
+}
```

Comment:
Is this necessary if PVOs are listed as owners of the PV? We could then just use `Owns(&kmmv1beta1.PreflightValidation{})`.

Reply:
yes, it is necessary, when deleting the PreflightOCP, the Preflight is also deleted by kubernetes. Changed the For to Owns

Reply:
One PreflightOCP owns exactly one Preflight